langfuse/client.py

```diff
@@ -358,7 +357,7 @@ def update(self, body: UpdateGeneration):
 
             def task():
                 try:
-                    new_body = body.copy(update={"generation_id": self.id})
+                    new_body = body.copy(update={"generation_id": self.id, "trace_id": self.trace_id})
```

Comment:
Why this change? It does not destroy anything but did it not work without?

Reply:
Before this change it was not actually updating the generation because it couldn't find which trace it is.